apps/extension/src/core/domains/app/handler.ts

```diff
@@ -132,7 +131,7 @@ export default class AppHandler extends ExtensionHandler {
         pair.lock()
 
         // we can now set up the auth secret
-        await this.stores.password.setPlaintextPassword(pass)
+        this.stores.password.setPassword(transformedPassword)
```

Comment:
For context, this is the implementation of `this.stores.password.setPlaintextPassword`:

```ts
  public async setPlaintextPassword(plaintextPw: string) {
    const pw = await this.transformPassword(plaintextPw)
    this.setPassword(pw)
  }
  ```
  
  We've already derived a transformedPassword, so why don't we just call `setPassword` and avoid running another 1s hash?